packages/ifpack2/src/Ifpack2_Details_OverlappingRowGraph_decl.hpp

```diff
@@ -118,8 +118,10 @@ class OverlappingRowGraph :
   //! The communicator over which the graph is distributed.
   virtual Teuchos::RCP<const Teuchos::Comm<int> > getComm () const;
 
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
   //! The graph's Node instance.
   virtual Teuchos::RCP<node_type> getNode () const;
```

Comment:
We should probably mark the function with `TPETRA_DEPRECATED` as well, though this doesn't matter as much for virtual methods where the base class' method has the marking.  Ditto for other instances in this PR.  I mention this in part because you do mark some functions with `TPETRA_DEPRECATED`, like `getOrCreateLocallyReplicatedTpetraVectorSpace` below.

Reply:
Done